### DISCUSSION TITLE
Not a cloud-agnostic stack

### DISCUSSION THREAD

**Discussion by cloudcompute** [stacksjs/stacks#1085] [General]
It could be framework agnostic but not cloud-native. it is not a cloud-agnostic stack. AWS vendor lockin.

Also, Vue-favored, why not React-based, don't know. This point does not matter much because both are completely open-source.

**Comment by chrisbbreuer** [comment]
AWS is our first cloud driver. GCP, Azure, Fly and others will join in later releases.

Our UI engine currently is Vue-flavored because our team personally/internally prefers using Vue. But the UI engine will eventually pivot into allowing for other drivers as well, and React will be one of them 👍 

_(There are a few other features that need to be ported in order to become compatible with our UI engine, e.g. our Library features, i.e. automatically building a component lib into a framework-agnostic web component lib, and a few other feats)_
